tests/cli/test_cli_regression_mol+mol.py

```diff
@@ -62,7 +62,7 @@ def test_train_quick(monkeypatch, data_path):
         "--show-individual-scores",
     ]
 
-    task_types = ["", "regression-mve", "regression-evidential"]
```

Comment:
If the task type is regression-evidential, we set the evidential-regularization to 0.2. If task type is regression-quantile, should we set alpha to a value?

Reply:
The default value for `alpha` is 0.1, so I think it is fine not testing the other values.